docs/content/users/install/ddev-installation.md

```diff
@@ -107,11 +105,19 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
     yay -S ddev-bin
     ```
 
+    ```bash
+    # One-time initialization of mkcert
+    mkcert -install
+    ```
```

Comment:
This can be in the same block as `yay` command. (I can't make a suggestion here.)